DZ1_var11.py

```diff
@@ -0,0 +1,97 @@
+# Вариант 11
+# Коэффициенты системы линейных уравнений заданы в виде прямоугольной матрицы.
+# С помощью допустимых преобразований привести систему к треугольному виду.
+# Найти количество строк, среднее арифметическое элементов которых, меньше заданной величины.
+import random
+
+ARITHMETIC_MEDIUM = 5
+
+
+def get_rnd_matrix(min_val, max_val, row, col):
+    return [[random.randint(min_val, max_val) for _ in range(col)] for _ in range(row)]
+
+
+def validate_matrix(matrix):
+    num_cols = len(matrix[0])
+    num_rows = len(matrix)
+
+    # verification of data
+    if num_rows > num_cols:
+        return False, 'number of rows is not equal to number of cols'
```

Comment:
and now your validation from main will work as True because you returned unempty tuple